components/QueryInfo/QueryInfo.tsx

```diff
@@ -0,0 +1,54 @@
+import { CheckCircleIcon, AlertFillIcon } from '@primer/octicons-react'
+import React from 'react'
+import { Spinner } from 'react-bootstrap'
+import styles from '../../scss/queryInfo.module.scss'
+
+type QueryInfo = {
+  data: any
+  loading: boolean
+  error: string
+  texts: {
+    loading: string
+    errorTitle: string
+  }
+  DataMessage?: () => JSX.Element
+}
+
+const QueryInfo = ({ data, loading, error, texts, DataMessage }: QueryInfo) => {
```

Comment:
1) Will these notification messages stay on the screen forever, or will they go away on their own?

2) Is it possible to close these notification messages that pop up? As in, is there an `X` button somewhere I can click to close the notification?

Reply:
It's not possible to close them and they'll remain forever on the screen. I think it's better to hide them after some seconds.

Reply:
Can we the message go away after a few seconds in this PR too?

Reply:
I think the only message that should go away after some seconds is the success one. The loading message go away after the data or error resolves. 

The error message is important and it should remain on screen until the admin switch to a different lesson/module/challenge to update/add (different issue in this [component](https://github.com/garageScript/c0d3-app/blob/7c98313c745a0997d49adda34102a79e0ee3d6ba/components/admin/lessons/AdminLessonInputs/AdminLessonInputs.tsx#L74-L84) that's related to this).

Reply:
Why is the error message notification important to keep on screen? We shouldn't rely on info from the notification message, that's only there to notify us that something went wrong, so I don't see a reason to keep it forever on the screen. A developer can gain more details by looking at the actual request(in network tab of devtools) that's not working as expected.

Reply:
Also, what if we want to implement this for features that are public-facing? It would be bad experience for users to have a popup notification appear and stay on screen forever until they switch pages